go.mod

```diff
@@ -8,4 +8,5 @@ require (
 	github.com/sethvargo/go-password v0.1.2
 	github.com/spf13/cobra v0.0.5
 	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
+	k8s.io/apimachinery v0.17.3
```

Comment:
This dependency should be removed. `go mod tidy` can fix it.

Reply:
Thank you for your help, it seems to be working fine now. I've submitted a new PR. 

I cannot @ you for some reason.